try:
    
    print(val)

except NameError:
    
    print("Value is not defined")
    
except:
    
    raise Exception(self)